internal/lib/lib.go

```diff
@@ -1959,19 +1959,52 @@ func IsChanClosed(ch <-chan struct{}) bool {
 	return false
 }
 
-func GetIPFromNode(node *v1.Node) (string, string) {
-	var nodeV4, nodeV6 string
```

Comment:
why did we remove this and return statement?

Reply:
it was just a choice to use named return values.
i have changed it back to improve readability.